### ISSUE TITLE
Who Maintains the Maintainers?

### ISSUE THREAD

**Issue by m4v15** [foundersandcoders/nazareth-recipes#55]
When Nazareth Repo's was but a fledgling repository wallowing in the murky depths of the founders and coders organisation, the question of who should be a maintainer seemed not too important and not too hard to choose - there were only really the 4 of us active on it at the time. However, since then we have grown to a size where who exactly gets to make big decisions should be decided in a transparent and fair manner. I don't know how this should be decided. But just feel as though the discussion needs to be had.

**Comment by Karyum** [comment]
@m4v15 maintainers should be people who live in Nazareth like me an @finnhodgkin and @matthewdking  and also finished the programme (*for good github workflow, funny that i said that but also goes to you @m4v15*).
so now you and @mattlub are not fit anymore to be maintainers i'm not usre you have the right anymore to be one since you guys currently live in London

**Comment by m4v15** [comment]
Hmmm I can see your argument but I would say as has been discussed here #47, Nazareth Recipes is about more than just being in Nazareth. But I would be open to other opinions. I think the most important thing for a maintainer is a calm head 🤙  🌵  🤕

**Comment by Karyum** [comment]
@m4v15 I am very calm and enthusiastic about this repo but when people are being a**holes towards me im not gonna take it.
It is literally personal just have a think with yourself, even @matthewdking  said he is being a d*ck towards but he still did it and would not do it to someone else

**Comment by Karyum** [comment]
So now i don't request for my omelette to be merged i also request for an apology

**Comment by m4v15** [comment]
Stop having this argument in about 20 different places jeeze.

But also, it's really not personal :heart:

**Comment by Karyum** [comment]
Merge my pr and everything would go back to normal

**Comment by Karyum** [comment]
also here 
https://github.com/foundersandcoders/nazareth-recipes/pull/33
you merged a PR and @ionush request a change and also @finnhodgkin was writing a comment how dare you talk about integrity and good github workflow?

**Comment by shiryz** [comment]
@m4v15 are you suggesting meta-maintainers for the repo, people who are impartial to this?

**Comment by m4v15** [comment]
God we could have all been doing productive work this evening guys

**Comment by m4v15** [comment]
I mean, not that this isn't important obvs.

@Karyum there's no way I could have known that @finnhodgkin was making a comment when I merged! Admittedly I should have I guess waited for @ionush's to approve but again like I say, the main thing is merging you're own PR. get over it pal.

**Comment by m4v15** [comment]
Also it was your cake recipe ffs! You were bugging me to merge that for _months_

**Comment by mattlub** [comment]
a couple of points:
- @Karyum I think maintainers should maintain until they choose to resign. According to current protocol having periodically inactive maintainers does not adversely affect process.
- I agree @matthewdking and @finnhodgkin should become maintainers, and am happy for that to happen. We probably need to decide how we do such things. I would say all active maintainers agreeing on it is sufficient for now until we have a better process. (@m4v15 ?)

**Comment by m4v15** [comment]
@mattlub Yes I would agree I think having inactive maintainers is not a problem as long as there are enough active ones to deal with upkeep. Also I would say being in Nazareth or not doesn't make a huge difference in terms of the jobs of a maintainer. A maintainer shouldn't have too much undue power anyway, they are just people who can do the final merge, so just need to be trustworthy that they won't merge anything unwanted into master, we don't need them to be anything else really.

I would agree @matthewdking and @finnhodgkin are more than suitable candidates for maintainers - but do they actually want to be? We are yet to here from them on this matter. Should we head-hunt potential maintainers and then offer them the role (as we are now) or should we allow them to apply for the role? I think maybe head hunting, as is happening now might be better.

**Comment by finnhodgkin** [comment]
For sure @matthewdking should be a maintainer, but I don't think I'm a suitable candidate _yet_. IMO as a minimum requirement maintainers should have at least one recipe merged to master.

**Comment by yvonne-liu** [comment]
I would like to point out the lack of diversity in your current and proposed maintainers.

**Comment by m4v15** [comment]
@yvonne-liu While I agree the lack of diversity in the current maintainer list is troubling, I would argue that it is not that we are overlooking potentially diverse candidates but that we simply don't have enough (any?) diverse candidates for the role. This is still a fault of Nazareth Recipes and we should be doing more to encourage participation from a wider community in order that we can in the future have a more diverse range of maintainers. I would say we definitely have not been actively discouraging it, it just appears we must do more to encourage it. 

Affirmative action (*the policy of favouring members of a disadvantaged group who suffer or have suffered from discrimination within a culture*)  is a highly controversial subject, with advocates and opponents from across the political spectrum and I would more edge towards Positive Action (*the promotion of people based on belonging to non majority identity groups in the workplace, educational institutions and positions in society, purportedly without prejudicing the criteria of selection by merit*) - that is to say, if we have qualified, diverse, candidates they should maybe be favoured over a similarly qualified non-diverse candidate, but we should not be choosing less qualified applicants who are simply more diverse than other, more qualified applicants, but also that we should be doing everything we can to ensure we get the qualified, diverse candidates. ❤️
